### ISSUE TITLE
Updating fonts for lessons in Ukrainian

### ISSUE THREAD

**Issue by olexandr-konovalov** [carpentries/varnish#120]
The initial discussion is in https://github.com/ukrainian-carpentries/git-novice/issues/1

As @zkamvar suggested that the change should be done in this repository, I am cross-posting it here. We would like to change the font at https://ukrainian-carpentries.github.io/git-novice/ to use any decent sans serif one. 

I am not an expert in fonts and CSS, but happy to make a PR if someone will confirm what to do (not sure how to test it either).

@zkamvar @froggleston I have downloaded Mulish.zip from https://fonts.google.com/specimen/Mulish. It gives 

```
This means all the styles are contained in these files:
  Mulish-VariableFont_wght.ttf
  Mulish-Italic-VariableFont_wght.ttf
  static/Mulish-ExtraLight.ttf
  static/Mulish-Light.ttf
  static/Mulish-Regular.ttf
  static/Mulish-Medium.ttf
  static/Mulish-SemiBold.ttf
  static/Mulish-Bold.ttf
  static/Mulish-ExtraBold.ttf
  static/Mulish-Black.ttf
  static/Mulish-ExtraLightItalic.ttf
  static/Mulish-LightItalic.ttf
  static/Mulish-Italic.ttf
  static/Mulish-MediumItalic.ttf
  static/Mulish-SemiBoldItalic.ttf
  static/Mulish-BoldItalic.ttf
  static/Mulish-ExtraBoldItalic.ttf
  static/Mulish-BlackItalic.ttf
```
Do I have to add all these `.ttf` and for each of them use https://transfonter.org/ to do conversions, to also add `.woff`, `.woff2`, `.svg`, and `.eot`, and also create CSS stylesheets and update [this file](https://github.com/carpentries/varnish/blob/1a863b61af402d1d244833e7dccb67894067b0e3/source/stylesheets/styles.css.scss#L46-L59) or also other CSS files? Do I need to add `static` subdirectory?

**Comment by astroDimitrios** [comment]
That looks right to me - you would need to update the files in the `source\fonts` directory in Varnish. I don't know that you would need to update any CSS. Try updating the files in the `source\fonts` dir first with the files from Mullish.zip and transfonter.org and see what happens!

I know it doesn't affect many but I notice Mullish doesn't have any Greek characters - I wonder if there is a font that covers all character sets? Don't know how big the Greek carpentries community is!

**Comment by olexandr-konovalov** [comment]
@zkamvar @froggleston @astroDimitrios thanks again - I've submitted https://github.com/carpentries/varnish/pull/133

**Comment by olexandr-konovalov** [comment]
Thanks @froggleston! What should I do to see this at https://ukrainian-carpentries.github.io/git-novice/ ? I have made a few updates to trigger builds, nothing happened. I went to https://github.com/ukrainian-carpentries/git-novice/actions and triggered some other builds manually in the usual order - again, nothing happened. Is there some other update coming elsewhere?

**Comment by froggleston** [comment]
Hi @olexandr-konovalov - your builds are using 1.0.4 and the font update is in 1.0.5.9000, the development version. This will be available once I've released 1.0.5, or you can manually set your varnish version to the main branch rather than the r-universe release in your config.yaml.

**Comment by olexandr-konovalov** [comment]
Thanks for the explanation @froggleston! It depends - when do you think 1.0.5 will be released?
